azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -63,9 +63,17 @@ def __init__(self, connection_string):
         :returns: Instance of the IoTHubRegistryManager object.
         :rtype: :class:`azure.iot.hub.IoTHubRegistryManager`
         """
-
         self.auth = ConnectionStringAuthentication(connection_string)
         self.protocol = protocol_client(self.auth, "https://" + self.auth["HostName"])
+        self.amqp_svc_client = iothub_amqp_client(
+            self.auth["HostName"], self.auth["SharedAccessKeyName"], self.auth["SharedAccessKey"]
+        )
+
+    def __del__(self):
```

Comment:
>def __del__(self): [](start = 4, length = 18)

Random unrelated note: __del__ does not exist on micropython.  Luckily we don't use this too much.